samples/suit/smp_transfer/README.rst

```diff
@@ -118,15 +118,15 @@ You can enable the external flash support by setting the following ``FILE_SUFFIX
 
 .. code-block:: console
 
-   west build -p -b nrf54h20dk/nrf54h20/cpuapp -- -DFILE_SUFFIX="extflash"
+   west build --pristine -b nrf54h20dk/nrf54h20/cpuapp -- -DFILE_SUFFIX="extflash"
```

Comment:
There are 80 other occurrences in the NCS where `-p` is used, but only here we're expanding it. 
What is the reason for this change?

Reply:
I thought that will be an improvement to make the difference more visible, in commands where we only build an updated image it is beneficial to not have a pristine build. Should I change it to `-p` for this to be consistent in NCS?

Reply:
Yes, this is a good change, but we should probably make it for all 80+ commands. I suggest you revert it here and we open a separate PR after the release.

Reply:
Yeah that will be better, thanks.